### PR TITLE
2575: Focus trap bottom action sheet

### PR DESCRIPTION
**Pull request by f1sh1918**
### Short description

Focus trap lib since `10.x` can not detect nodes from a portal (bottom action sheet) and will throw an error (opening feedback modal)

### Proposed changes

- added a legacy displayCheck for finding the node in the portal

### Resolved issues

Fixes: #2575 

---

### CODE REVIEW

File: web/src/components/Modal.tsx [hunk 1]
```diff
@@ -62,9 +62,9 @@ const Modal = ({ title, closeModal, children, direction, wrapInPortal = false }:
 
     return () => layoutElement?.setAttribute('aria-hidden', 'false')
   }, [])
-
+  // display check option is needed for portals - https://github.com/focus-trap/tabbable/blob/master/CHANGELOG.md#532
```
Comment (steffenkleinle) [inline web/src/components/Modal.tsx:65]: I think you linked the wrong release, shouldnt it be https://github.com/focus-trap/tabbable/blob/master/CHANGELOG.md#600?

It says
> restore old (incorrect) behavior

Should we investigate a better fix?

**Review by steffenkleinle — APPROVED** [review]
Tested on firefox

File: web/src/components/Modal.tsx (hunk 1 above)
Comment (f1sh1918) [reply web/src/components/Modal.tsx:65]: Gonna adjust the link. Yes I can create an New issue for that. I think we should use this fix as a workaround

**Review by ztefanie — APPROVED** [review]
Looks good.

**Pull request merged**